pkg/client/retry.go

```diff
@@ -18,12 +18,29 @@ package client
 
 import (
 	"context"
+	"time"
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/retry"
 	kbclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// MinuteBackoff is a retry.DefaultBackoff that retries for at least 1 minute (60000ms)
+var MinuteBackoff = func() wait.Backoff {
+	mb := retry.DefaultBackoff
+	// TotalDuration = 0ms + 10ms + 50ms + 250ms + 1250ms + 6250ms + 31250ms + 60000ms = 99,060 ms > 1 minute
```

Comment:
instead of saying its greater than 1 minute, what about saying its less than 2 minutes?

Reply:
@mateusoliveira43 1 minute is the cap defined here, so that last iteration will happen at 60s rather than something larger than 2 minutes (that 60000ms should actually be 31250*5ms, but again since the last iteration would happen at a time greater than the defined cap of 1 minute, it doesn't matter -- last retry will be 60s after initial call, and if it's still not ready then, the original error message (NotFound) will be returned.

Reply:
Understood, thanks